R/batch-api.R

```diff
@@ -37,8 +37,18 @@ BatchUtilities <- R6::R6Class(
             envFile,
             readToken,
             config$endpointSuffix)
-        resourceFiles <-
-          list(rAzureBatch::createResourceFile(url = envFileUrl, fileName = envFile))
+
+        environmentResourceFile <-
+          rAzureBatch::createResourceFile(filePath = envFile, httpUrl = envFileUrl)
+
+        if (is.null(resourceFiles))
+        {
+          resourceFiles <- append(resourceFiles, environmentResourceFile)
+        }
+        else {
+          resourceFiles <-
+            list(environmentResourceFile)
+        }
```

Comment:
Shouldn't this be reversed if null set to list else append to existing list?

Reply:
Will fix this.